copier.yml

```diff
@@ -87,6 +87,20 @@ setup_github:
   type: bool
   help: "Setup GitHub repository (requires gh CLI)?"
   default: no
+  when: "{{ initial_commit }}"
+
+github_token:
+  type: str
+  secret: true
+  help: "GitHub Token with `repo` and `workflow` scopes (optional)"
+  when: "{{ setup_github }}"
+  default: ""
+
+github_app_installation_ids:
+  type: str
+  help: "Comma separated list of installation IDs of GitHub Apps you want to have installed (optional) (presumably Renovate and pre-commit.ci). The IDs could be found from the url of the 'Configure' buttons in https://github.com/settings/installations. e.g. https://github.com/organizations/<Organization-name>/settings/installations/<ID>. See https://stackoverflow.com/questions/74462420/where-can-we-find-github-apps-installation-id for further details."
```

Comment:
That's not very user friendly, I'd rather go in GitHub and install them in the UI.

Reply:
This is optional and if an empty string is given nothing should happen.
I don't fully understand the features of Copier, but maybe this could be a hidden option if the question is annoying. should I do that?

Reply:
I'm not sure I follow what you suggesting, so maybe I'm missing something, but generally speaking I'm strongly -1 on the idea of this change, as I think it would be too brittle to maintain: entering GitHub token and IDs of integrations + using the REST API directly

Reply:
This question is meant to be used with [Default settings](https://copier.readthedocs.io/en/stable/settings/#trusted-locations). I want to focus very much on automation, as I make packages as often as once a month. 

I assume you are simply refusing this because you have never tried it, and I hope you could try #681 once (it's outdated against main branch but is still working☺️) and experience it for yourself. (note that this branch is still WIP)